### PR TITLE
quiet compiler, fix some whitespace

### PR DESCRIPTION
**Pull request by sethalves**

### CODE REVIEW

File: libraries/render-utils/src/Model.cpp [hunk 1]
```diff
@@ -2106,16 +2106,16 @@ void Model::renderPart(RenderArgs* args, int meshIndex, int partIndex, bool tran
     const FBXMesh& mesh = geometry.meshes.at(meshIndex);
     const MeshState& state = _meshStates.at(meshIndex);
     
-    bool translucentMesh = translucent; // networkMesh.getTranslucentPartCount(mesh) == networkMesh.parts.size();
+    // bool translucentMesh = translucent; // networkMesh.getTranslucentPartCount(mesh) == networkMesh.parts.size();
```
Comment (ZappoMan) [inline libraries/render-utils/src/Model.cpp]: what's up with this change?

File: libraries/render-utils/src/Model.cpp [hunk 2]
```diff
@@ -2106,16 +2106,16 @@ void Model::renderPart(RenderArgs* args, int meshIndex, int partIndex, bool tran
...
     bool hasTangents = !mesh.tangents.isEmpty();
     bool hasSpecular = mesh.hasSpecularTexture();
     bool hasLightmap = mesh.hasEmissiveTexture();
     bool isSkinned = state.clusterMatrices.size() > 1;
     bool wireframe = isWireframe();
     
-    if (wireframe) {
-        translucentMesh = hasTangents = hasSpecular = hasLightmap = isSkinned = false;
-    }
+    // if (wireframe) {
```
Comment (ZappoMan) [inline libraries/render-utils/src/Model.cpp]: and this?

**Comment by hifi-gustavo** [comment]
Build console output for commit: 983ae5b2f02e3f1509bbd39a490b3e48d8c95b4c available at: https://gist.github.com/af79aed964cce372bb56<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5086.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5086.dmg

File: libraries/render-utils/src/Model.cpp (hunk 1 above)
Comment (sethalves) [inline libraries/render-utils/src/Model.cpp]: compiler complains that translucentMesh is initialized but unused.

File: libraries/render-utils/src/Model.cpp (hunk 2 above)
Comment (sethalves) [inline libraries/render-utils/src/Model.cpp]: maybe the call to pickPrograms was meant to have translucentMesh as an argument rather than translucent?

Comment (sethalves) [inline libraries/render-utils/src/Model.cpp]: looking at this harder, I have to believe that was the intention.  I've adjusted the pr.

**Comment by hifi-gustavo** [comment]
Build console output for commit: fb7a0e72196ffc5dab13b55eedc2306eccd3d87e available at: https://gist.github.com/af79aed964cce372bb56<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5086.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5086.dmg

**Pull request merged**